Lib/dis.py

```diff
@@ -515,13 +515,24 @@ def _disassemble_str(source, **kwargs):
 
 disco = disassemble                     # XXX For backwards compatibility
 
+
+# Rely on C `int` being 32 bits for oparg
+INT_BITS = 32
+# Maximum value for a c int
+INT_MAX = 2 ** (INT_BITS - 1)
```

Comment:
```suggestion
_INT_BITS = 32
# Maximum value for a c int
_INT_MAX = 2 ** (_INT_BITS - 1)
```

Let's make these private so that we don't inadvertently create a new public API